worker/awsCertlint/awsCertlint.go

```diff
@@ -0,0 +1,167 @@
+	tmp, err := ioutil.TempFile("", "awslabs-certlint")
+	if err != nil {
+		return nil, fmt.Errorf("error creating temp dir", cert.ID)
+	}
+	defer os.Remove(tmp.Name())
+	x509Cert, err := cert.ToX509()
+	if err != nil {
+		return nil, fmt.Errorf("error converting to x509.Certificate", cert.ID)
+	}
+	if err := ioutil.WriteFile(tmp.Name(), x509Cert.Raw, 0644); err != nil {
+		return nil, fmt.Errorf("error writing x509.Certificate to temp file, err=%v", err)
+	}
+
+	// Run certlint over certificate
+	cmd := exec.Command("ruby", "-I", "lib:ext", "bin/certlint", tmp.Name())
+	cmd.Dir = certlintDirectory
+
+	var stdout bytes.Buffer
+	cmd.Stdout = &stdout
+	var stderr bytes.Buffer
+	cmd.Stderr = &stderr
+
+	// attach stdout/stderr
+	if err := cmd.Run(); err != nil {
+		return nil, fmt.Errorf("error running awslabs/certlint, err=%v, out=%q", cert.ID, strings.TrimSpace(stderr.String()))
+	}
+
+	return e.parseResponse(stdout)
+}
+
+
+// From: https://github.com/awslabs/certlint#output
+//
+// * B: Bug. Your certificate has a feature not handled by certlint.
+// * I: Information.  These are purely informational; no action is needed.
+// * N: Notice.  These are items known to cause issues with one or more implementations of certificate processing but are not errors according to the standard.
+// * W: Warning.  These are issues where a standard recommends differently but the standard uses terms such as "SHOULD" or "MAY".
+// * E: Error.  These are issues where the certificate is not compliant with the standard.
+// * F: Fatal Error.  These errors are fatal to the checks and prevent most further checks from being executed.  These are extremely bad errors.
+func (e eval) parseResponse(resp bytes.Buffer) (*Result, error) {
+	out := &Result{}
+
+	r := bufio.NewScanner(&resp)
+	for r.Scan() {
+		line := strings.TrimSpace(r.Text())
+		if line == "" {
+			continue
+		}
+
+		// Match first letter of each line, which signifies its type.
+		switch line[0] {
+		case 'B':
+			out.Bugs = append(out.Bugs, strings.TrimSpace(line[2:]))
+		case 'I':
+			out.Informational = append(out.Informational, strings.TrimSpace(line[2:]))
+		case 'N':
+			out.Notices = append(out.Notices, strings.TrimSpace(line[2:]))
+		case 'W':
+			out.Warnings = append(out.Warnings, strings.TrimSpace(line[2:]))
+		case 'E':
+			out.Errors = append(out.Errors, strings.TrimSpace(line[2:]))
+		case 'F':
+			out.FatalErrors = append(out.FatalErrors, strings.TrimSpace(line[2:]))
+		}
+	}
+	return out, nil
+}
+
+// AnalysisPrinter outputs the results from awslabs/certlint over a given certificate
+func (e eval) AnalysisPrinter(input []byte, printAll interface{}) (results []string, err error) {
+	var result Result
+	if err := json.Unmarshal(input, &result); err != nil {
+		return nil, fmt.Errorf("awsCertlint Worker: failed to parse results: err=%v", err)
+	}
+
+	// Build results for webview
+	results = append(results, "* awslabs/certlint Results")
```

Comment:
nit: remove "Results"
It would be nice to have a summary on the first line, such as `* awslabs/certlint: no issues found` or `1 warning found`